app/src/main/res/layout/app_bar.xml

```diff
@@ -28,15 +28,15 @@
         <View
             android:layout_width="1dp"
             android:layout_height="match_parent"
-            android:background="?attr/colorPrimary" />
+            android:background="?attr/white" />
 
         <ImageButton
             android:id="@+id/action_schedule"
             android:layout_width="0dp"
             android:layout_height="match_parent"
             android:layout_margin="10dp"
             android:layout_weight="1"
-            android:background="@color/white"
+            android:background="@color/colorPrimary"
```

Comment:
Same as above, this parameter value will give the background the same color as the icon, therefore making it to be invisible. You can just leave it as unchanged